ssl/ssl_aead_ctx.cc

```diff
@@ -428,4 +429,83 @@ bool SSLAEADContext::GetIV(const uint8_t **out_iv, size_t *out_iv_len) const {
          EVP_AEAD_CTX_get_iv(ctx_.get(), out_iv, out_iv_len);
 }
 
+#define SSLAEADCONTEXT_SERDE_VERSION 1
```

Comment:
serde is the name of a Rust crate, but in this case I was using it as shorthand for SErialization and DEserialization since it was a common acronym I was already use to using from the SDK side. I'm open to spelling this out / picking a direction like `SSLAEADCONTEXT_SERIALIZATION_VERSION` if anyone feels strongly.

Reply:
TIL